pkg/oc/cli/admin/diagnostics/diagnostics/cluster/scc.go

```diff
@@ -59,36 +57,35 @@ func (d *SCC) Check() types.DiagnosticResult {
 		InfraNamespace: bootstrappolicy.DefaultOpenShiftInfraNamespace,
 	}
 
-	changedSCCs, err := reconcileOptions.ChangedSCCs()
+	newSCCs, changedSCCs, err := reconcileOptions.ChangedSCCs()
 	if err != nil {
 		r.Error("CSD1000", err, fmt.Sprintf("Error inspecting SCCs: %v", err))
 		return r
 	}
+	for _, newSCC := range newSCCs {
+		r.Error("CSD1001", nil, fmt.Sprintf("scc/%s is missing.\n\nUse the `oc adm policy reconcile-sccs` command to recreate sccs.", newSCC.Name))
+	}
 	changedSCCNames := map[string]bool{}
 	for _, changedSCC := range changedSCCs {
-		_, err := d.SCCClient.Get(changedSCC.Name, metav1.GetOptions{})
-		if kerrs.IsNotFound(err) {
-			r.Error("CSD1001", nil, fmt.Sprintf("scc/%s is missing.\n\nUse the `oc adm policy reconcile-sccs` command to recreate sccs.", changedSCC.Name))
-			continue
-		}
-		if err != nil {
-			r.Error("CSD1002", err, fmt.Sprintf("Unable to get scc/%s: %v", changedSCC.Name, err))
-			continue
-		}
 		r.Warn("CSD1003", nil, fmt.Sprintf("scc/%s will be reconciled. Use the `oc adm policy reconcile-sccs` command to check sccs.", changedSCC.Name))
 		changedSCCNames[changedSCC.Name] = true
 	}
 
 	// Including non-additive SCCs, but output messages with debug level.
 	reconcileOptions.Union = false
-	changedSCCs, err = reconcileOptions.ChangedSCCs()
+	newSCCsUnion, changedSCCsUnion, err := reconcileOptions.ChangedSCCs()
 	if err != nil {
 		r.Error("CSD1000", err, fmt.Sprintf("Error inspecting SCCs: %v", err))
 		return r
 	}
-	for _, changedSCC := range changedSCCs {
+
+	defaultsErrMsgFmt := "scc/%s does not match defaults. Use the `oc adm policy reconcile-sccs --additive-only=false` command to check sccs."
+	for _, newSCC := range newSCCsUnion {
+		r.Debug("CSD1004", fmt.Sprintf(defaultsErrMsgFmt, newSCC.Name))
+	}
+	for _, changedSCC := range changedSCCsUnion {
 		if !changedSCCNames[changedSCC.Name] {
```

Comment:
Is this `if` still required?

Reply:
I believe it still is. The help says:
```
--additive-only=true: If true, preserves extra users, groups, labels and annotations in the SCC as well as
existing priorities.
```
which means that some SCCs could still be changed in a way that they don't appear in the default search.

Reply:
What's more puzzling to me is why the message should appear all the time for the SCCs to be created, since they should be returned no matter the option.